src/components/Metrics/Metrics.container.ts

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux'
+import { RootState } from 'modules/common/types'
+import { getCurrentProject } from 'modules/project/selectors'
+import { EMPTY_SCENE_METRICS } from 'modules/scene/constants'
+import { MapDispatch, MapDispatchProps, MapStateProps } from './Metrics.types'
+import Metrics from './Metrics'
+import { getCurrentScene } from 'modules/scene/selectors'
+
+const mapState = (state: RootState): MapStateProps => {
+  const currentProject = getCurrentProject(state)
+  const currentScene = getCurrentScene(state)
+  if (currentProject && currentScene) {
+    return {
+      parcels: currentProject.parcels.length,
```

Comment:
Can we add a `TODO` here for a future change? parcels is meant to be the owned blockchain LAND for this scene, but it might not exist if you just created the project.
No need to change it here